test/test_helper.rb

```diff
@@ -70,3 +70,18 @@ def parsed_data
     end
   end
 end
+
+def mock_platform_api
+  require 'securerandom'
+
+  @url = 'http://localhost'
+  @b64_identity = '1234abcd'
+  @api = HostInventoryAPI.new(@account, @url, @b64_identity)
+  @connection = mock('faraday_connection')
+  @system_profile_response = OpenStruct.new(body: {
+    results: [{
+      id: "MOCK_INVENTORY_HOST_ID_#{SecureRandom.uuid}",
```

Comment:
@akofink Do you have an idea how to mock it so that when `Host.host_in_inventory` gets called it just returns a struct with whatever id it gets passed? Mocha doesn't seem to be able to take a block to use arguments passed. :/ Here I'm trying to mock the API, but it could be higher up as well.

Reply:
Like this?

```rb
Host.stubs(:host_in_inventory).returns(...)
```

Reply:
Yeah, but it can't take any argument. something like:
```ruby
Host.stubs(:host_in_inventory) do |host_id|
  ...
end
```
So the struct returned has the `host_id` as `id`

Reply:
I tried it with just redefining the method with `def Host.host_in_inventory` in the `setup`s, but that doesn't seem right.